homeassistant/components/tplink/sensor.py

```diff
@@ -144,21 +143,6 @@ class TPLinkSensorEntity(CoordinatedTPLinkFeatureEntity, SensorEntity):
 
     entity_description: TPLinkSensorEntityDescription
 
-    def __init__(
```

Comment:
No need for `__init__`s for `sensor` and `switch` as parent's `async_added_to_hass` takes care of updating the state.